testing/scenario_app/lib/src/locale_initialization.dart

```diff
@@ -134,8 +137,14 @@ class LocaleInitialization extends Scenario {
         childrenInTraversalOrder: Int32List(0),
         childrenInHitTestOrder: Int32List(0),
         additionalActions: Int32List(0),
-      )).build()
-    );
+      );
+
+    final SemanticsUpdate semanticsUpdate = semanticsUpdateBuilder.build();
+
+    for (final FlutterView view in dispatcher.views) {
+      view.updateSemantics(semanticsUpdate);
+    }
```

Comment:
In the other tests you online dispatched it to the first view, here its going to all views? Why this difference?

FWIW, choosing the first one everywhere is probably good enough.